Fix Declaration emit issue for CommonJS export

The updated taskQueue is using CommonJS exports, which are not legal with ES6 format. Hence, we see the error when compiling core in TS for Docs (reproduce by running `npm run docs`)

```
Error: lib/common/suite.js:81:24 - error TS9006: Declaration emit for this file requires using private name 'TaskQueue' from module '"/home/vipulgupta2048/work/leviathan/core/lib/common/taskQueue"'. An explicit type annotation may unblock declaration emit.

module.exports = class Suite {
                          ~~~~~

Error: lib/common/suite.js:81:24 - error TS9006: Declaration emit for this file requires using private name 'Teardown' from module '"/home/vipulgupta2048/work/leviathan/core/lib/common/taskQueue"'. An explicit type annotation may unblock declaration emit.

module.exports = class Suite {
                          ~~~~~
```


This is an edge case since Core is written in JS + TS, but the docs need to compile all code to TS in order to generate docs and that's when this error comes. Reference: https://stackoverflow.com/questions/60009092/declaration-will-not-emit-due-to-private-name-usage

--- a/core/lib/common/taskQueue.js
+++ b/core/lib/common/taskQueue.js
@@ -127,6 +127,13 @@ class Teardown extends TaskQueue {
 }
 
 module.exports = {
+	/**
+	 * @type TaskQueue
+	 */
 	Setup: TaskQueue,
+
+	/**
+	 * @type Teardown
+	 */
 	Teardown: Teardown,
 };

